spec/index.html

```diff
@@ -1233,9 +1235,17 @@ <h3>Escape Sequences</h3>
 
     <ul>
       <li>
-        <p><em id="numeric">numeric escape sequences</em> represent <a data-cite="I18N-GLOSSARY#dfn-code-point">Unicode code points</a>:</p>
-
-        <table id="tab-uchar">
+        <p>
+          A <dfn>numeric escape sequence</dfn> represents the value of
+          a <a data-cite="I18N-GLOSSARY#dfn-code-point">Unicode code point</a>:
+        </p>
+        <p>
+          A <a>numeric escape sequence</a> MUST NOT produce a code point value
+          in the the range <code class="codepoint">U+D800</code> to <code class="codepoint">U+DFFF</code>,
+          which is the range for 
```

Comment:
```suggestion
          inclusive, which is the range for 
```

Reply:
The table, which is the defintion, makes it clear.

Reply:
All other inclusive ranges in this document include the word "inclusive". I don't see what is saved by not including it here. The fact that this two row table includes the word in both rows makes its lack here even more odd.

Reply:
There is one other use of "inclusive".

The Media Type registration - which is not part of this PR - does not use it.

(Personally, I'd take them out because Unicode does not use the word.)

Reply:
I think consistency is better than inconsistency. More of these range statements included "inclusive" than did not, so that's the direction I went. Given that it's not used in other specs that discuss Unicode code point ranges, it's probably OK to omit, but I would then do so in all cases. A single statement somewhere (e.g., in a "conventions used in this document" or similar section) saying that all Unicode code point ranges mentioned are inclusive might still be reasonable, to guide and hopefully prevent misunderstandings by future readers.

Reply:
Changed to say "inclusive" on first use, then not used.